src/ZXing.Net.Maui/ZXing.Net.MAUI/ICameraView.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System;
 using Microsoft.Maui;
 using Microsoft.Maui.Graphics;
-using System;
```

Comment:
Should we fix this in the commit that added ZXing in?

Reply:
Extracted to a separate commit.